packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -103,6 +110,11 @@ export {
   StaticQuery,
   useStaticQuery,
   prefetchPathname,
+  Slice,
+  // only for internal use
+  SlicesContext,
+  SlicesMapContext,
+  SlicesPropsContext,
```

Comment:
we should just not put those things in public API point and keep it in internals